jazz_test-lambda/index.js

```diff
@@ -0,0 +1,122 @@
+// =========================================================================
+// Copyright ©  2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+Nodejs Template Project
+@author:
+@version: 1.0
+ **/
+
+const errorHandlerModule = require("./components/error-handler.js"); //Import the error codes module.
+const responseObj = require("./components/response.js"); //Import the response module.
+const configObj = require("./components/config.js"); //Import the environment data.
+const logger = require("./components/logger.js"); //Import the logging module.
+const utils = require("./components/utils.js");
+const aws = require('aws-sdk');
+const validateARN = utils.validateARN;
+const execStatus = utils.execStatus();
+
+var handler = (event, context, cb) => {
+
+  'use strict';
```

Comment:
Why is "use strict" only within the function? Make this as the first statement in the file.

Reply:
jenkins build is giving an error when using strick global scope
was asking to use it in functional scope -

Reply:
Most of our services have it as first statement and we haven't seen any issues with it. (eg: https://github.com/tmobile/jazz/blob/master/jazz_create-serverless-service/index.js). The error might be for some other reason, please look into it.

Reply:
Add a .jshintrc file with the following content: `{ "esversion":6,
  "node": true }`